front/src/app/shared/api-service/api.service.ts

```diff
@@ -26,4 +26,7 @@ export class ApiService {
   renameTaskGroup(id: string, title: string): Observable<NewTaskList> {
     return this.http.put<NewTaskList>(environment.apiUrl + `/list/${id}`, { list: { id: id, title: title }});
   }
+  deleteTaskList(id: string): Observable<null> {
+    return this.http.delete<null>(environment.apiUrl + `/list/${id}`);
+  }

```

Comment:
in this case `Observable<void>` has to be used (`null` is quite a specific value that represents nothing, but still 😅 )